tests/fixtures/pipeline_objects.py

```diff
@@ -500,13 +501,16 @@ def make_datastore(
 
                     ds.wcs.provenance = prov
                     ds.wcs.sources = ds.sources
+                    # make sure this is saved to the archive as well
+                    ds.wcs.save(verify_md5=False, overwrite=True) #copied this from above in PSF
```

Comment:
I think overwrite is not strictly needed (the default is to allow overwrite, yes?). 
As for `verify_md5`, I think that just saves some runtime but doesn't really matter either way.